File: aoc2015/src/test/java/hu/wolfman/aoc2015/Day02Test.java
package hu.wolfman.aoc2015;

import org.junit.jupiter.api.Test;

import java.util.Collections;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class Day02Test {

    @Test
    public void testPartOne() {
        Day day = new Day02();
        assertEquals(58, day.solvePartOne(Collections.singletonList("2x3x4")));
        assertEquals(43, day.solvePartOne(Collections.singletonList("1x1x10")));
    }

    @Test
    public void testPartTwo() {
        Day day = new Day02();
        assertEquals(34, day.solvePartTwo(Collections.singletonList("2x3x4")));
        assertEquals(14, day.solvePartTwo(Collections.singletonList("1x1x10")));
    }
}

File: src/test/kotlin/Day02Test.kt
import kotlin.test.Test
import kotlin.test.assertEquals

class Day02Test {
    @Test
    fun testDay02() {
        val list = readInput("day02test")
        assertEquals(150, Day02.solvePartOne(list))
        assertEquals(900, Day02.solvePartTwo(list))
    }
}